apps/files/src/components/FilesAppBar.vue

```diff
@@ -203,6 +203,9 @@ export default {
     },
     hasFreeSpace () {
       return this.quota.free > 0 || this.currentFolder.permissions.indexOf('M') >= 0
+    },
+    isIE () {
+      return !!window.MSInputMethodContext && !!document.documentMode
```

Comment:
shouldn't we use tools for that ? in core we used to have the npm package "bowser" for IE checks.

Reply:
Moved ie check into plugins so we can use it all over Phoenix.